syncstorage/db.go

```diff
@@ -217,23 +219,22 @@ func NewDB(path string, conf *Config) (*DB, error) {
   work is handled by private functions.
 */
 
-// LastModified gets the database modified time
-func (d *DB) LastModified() (modified int, err error) {
+// LastModified returns the top level last modified timestamp
+func (d *DB) LastModified() (int, error) {
 	d.Lock()
 	defer d.Unlock()
 
-	var m sql.NullInt64
+	lastMod, err := getKey(d.db, STORAGE_LAST_MODIFIED)
+	if lastMod == "" || err != nil {
+		return 0, err
```

Comment:
After thinking about this a bit more, I might leave this code as is. The `lastMod == ""` condition will only happen when: 

- the database is new. No data exists so `0, nil` is a proper response 
- the database is being migrated (before this PR). The next POST, PUT, DELETE will synchronize things

In the case of migration: 

- there's only THREE gosync users ;) It's probably better to leave the code as is and migrate those people
- it might not matter, as the storage level ts is only returned on `info/*` endpoints. 

@mhammond what would happen on the client side if GET info/collections the last-modified timestamp was totally wrong (0.00)? Would clients freak out? This would be fixed on the next POST, PUT, DELETE to the database.